src/TransientFETools/Transient2ndOrderFEOperators.jl

```diff
@@ -0,0 +1,188 @@
+"""
+Transient FE operator that is defined by a 2nd order transient Weak form
+"""
+struct Transient2ndOrderFEOperatorFromWeakForm{C} <: TransientFEOperator{C}
+  res::Function
+  jac::Function
+  jac_t::Function
+  jac_tt::Function
+  assem_t::Assembler
+  trial
+  trial_t
+  trial_tt
+  test::FESpace
+end
```

Comment:
We could consider here a tuple of Jacobians (N+1) and no need to replicate. Idem with trials.

Reply:
Done